### PR TITLE
Create Observing_Anyons_2020

### PR DESCRIPTION
**Pull request by vfatemi**
Webpage for the Observing Anyons 2020 mini-conference.

### CODE REVIEW

**Comment by vfatemi** [comment]
- [ ] Confirm date and times of conference
- [ ] Add affiliation and link webpages of panelists
- [ ] Title, abstract, author list for each talk
- [ ] Schedule Zoom event
- [ ] Embed Zoom registration

**Comment by vfatemi** [comment]
Temporarily added paper titles and paper links in lieu of actual seminar information.

File: Observing_Anyons_2020 [hunk 1]
```diff
@@ -0,0 +1,41 @@
+# Observing Anyons 2020, mini-conference
+
+**October 5, 2020**  
+9:30am to 12:30pm EDT ([view in your time zone](https://everytimezone.com/s/4ab0cea5))
+
+Observing Ayons 2020 is a mini-confernece that convenes experts to present and discuss the latest advances on observing anyons in nature. 
+This year two experiments efforts claimed the first observation of fractional statistics in fractional quantum Hall devices.
```
Comment (babaks77) [inline Observing_Anyons_2020:7]: experimental efforts ...

**Comment by babaks77** [comment]
I think the webpage looks good to start advertising.

**Comment by vfatemi** [comment]
Fixed some words.

**Comment by akhmerov** [comment]
> I think the webpage looks good to start advertising.

We should probably merge this first :P 

Let me know when it's time to review and merge.

**Review by akhmerov — CHANGES_REQUESTED** [review]
We need to link the event from the header and the table of contents

File: _sidebar.md [hunk 2]
```diff
@@ -1,6 +1,7 @@
 * [Home](/)
 * Current and planned events
     * [Long Range Colloquium](long_range_colloquium.md)
+    * [Observing_Anyons_2020](Observing_Anyons_2020.md)
```
Comment (akhmerov) [inline _sidebar.md:4]:
```suggestion
    * [Observing Anyons 2020](Observing_Anyons_2020.md)
```

**Comment by vfatemi** [comment]
> We need to link the event from the header and the table of contents

Did these just now, but I have to depart for a bit. I haven't yet sorted out the "test changes to the markdown content locally" mechanism.

**Pull request merged**